cirq/sim/clifford/act_on_stabilizer_ch_form_args.py

```diff
@@ -34,16 +34,41 @@ class ActOnStabilizerCHFormArgs:
     Measurements are currently not supported on this object.
     """
 
-    def __init__(self, state: StabilizerStateChForm, axes: Iterable[int]):
+    def __init__(
+        self,
+        state: StabilizerStateChForm,
+        axes: Iterable[int],
+        prng: np.random.RandomState,
```

Comment:
This could be default `np.random.RandomState()` - why wouldn't we set it?

Reply:
Honestly, I was just following the example set in `ActOnStateVectorArgs`. I think the reasoning is that, except for tests, the simulator that is using this object should be supplying the prng. So making this arg optional may lead to easier mistake where the seeded prng of the simulator is not being passed through.

Reply:
Sounds good!